test/setup/iam.tf

```diff
@@ -31,14 +38,33 @@ resource "google_service_account" "int_test" {
   display_name = "kms-int-test"
 }
 
+resource "google_organization_iam_member" "org_admins_group" {
+  for_each = toset(local.int_org_required_roles)
+  org_id   = var.org_id
+  role     = each.value
+  member   = "serviceAccount:${google_service_account.int_test.email}"
+}
+
 resource "google_project_iam_member" "int_test" {
-  count = length(local.int_required_roles)
+  for_each = toset(local.int_required_roles)
 
   project = module.project_ci_kms.project_id
-  role    = local.int_required_roles[count.index]
+  role    = each.value
   member  = "serviceAccount:${google_service_account.int_test.email}"
 }
 
+resource "google_billing_account_iam_member" "tf_billing_user" {
+  billing_account_id = var.billing_account
+  role               = "roles/billing.admin"
+  member             = "serviceAccount:${google_service_account.int_test.email}"
+}
+
+resource "google_billing_account_iam_member" "billing_account_log_config" {
+  billing_account_id = var.billing_account
+  role               = "roles/logging.configWriter"
```

Comment:
for my own understanding, what was this used for?

Reply:
`test/setup` configures all the resources, including permissions, for the CI to run.
Because in this autokey module we are creating a new folder and project (and this behavior is new for this repo), we need to add additional permissions into the SA in order to attach CFT billing account to the new project - otherwise we would receive a 403 because the SA used by CI doesn't have enough permissions yet.